apps/editing-toolkit/editing-toolkit-plugin/editor-gutenboarding-launch/index.ts

```diff
@@ -50,15 +63,15 @@ function updateEditor() {
 		}
 		clearInterval( awaitSettingsBar );
 
-		const isMobile = window.innerWidth < 782;
+		const isMobileViewport = window.innerWidth < 782;
```

Comment:
Magic number. Would be good if we can assign this to a named const somewhere (even if it's in the same file).

Reply:
Good point! @yansern is there a specif reason why you came up with `782`? Just to help to name this variable

Reply:
782 is $break-medium from `@wordpress/base-styles/_breakpoints.scss`. 

From package `@automattic/viewport`, there is no such viewpoint width (and also `@automattic/viewport` behaves differently, like a hook? so didn't want to complicate what is just simple JS code here).